src/main/java/com/bobocode/hw22nasapicturesstealer/model/Camera.java

```diff
@@ -0,0 +1,29 @@
+package com.bobocode.hw22nasapicturesstealer.model;
```

Comment:
I think it's better to store it separately in it's own package, something like `dao` package.